ui/raidboss/data/04-sb/eureka/eureka_hydatos.js

```diff
@@ -1075,108 +1075,479 @@
+        'Cloud to Ground': 'Attaque fulminante',
+        'Cube Form': 'Cube Form', // FIXME
+        'Eidos': 'Sarva',
+        'Elemental Magicks': 'Magie élémentaire',
+        'Elemental Shift': 'Changement d\'élément',
+        'Execration': 'Exécration',
+        'Explosion Enrage': 'Explosion Enrage', // FIXME
+        'Explosive Impulse': 'Impulsion explosive',
+        'Flare Star': 'Astre flamboyant',
+        'For Honor': 'Carnage martial',
+        'Holy': 'Miracle',
+        'Hostile Aspect': 'Onde polarisée',
+        'Impact Stream': 'Courant d\'impact',
+        'Ivory Palm': 'paume d\'ivoire',
+        'Lancing Bolt': 'Lance fulminante',
+        'Lateral Zantetsuken': 'Hen Zantetsuken',
+        'Legendary Geas': 'Tuerie spectrale',
+        'Legendary Imbas': 'Fiel spectral',
+        'Legendcarver': 'Taillade spectrale',
+        'Legendspinner': 'Spirale spectrale',
+        'Levinwhorl': 'Vortex de foudre',
+        'Medusa Javelin': 'Javelot de Méduse',
+        'Meteor': 'Météore',
+        'Mourning Star': 'Étoile du matin',
+        'Mythcall': 'Invitation fantasmagorique',
+        'Orb x5': 'Orb x5', // FIXME
+        'Ozma Ability': 'Ozma Ability', // FIXME
+        'Piercing Dark': 'Lance des ténèbres',
+        'Piercing Light': 'Lance de lumière',
+        'Pitfall': 'Embûche',
+        'Pyramid Form': 'Pyramid Form', // FIXME
+        'Random Form': 'Random Form', // FIXME
+        'Random Shade': 'Random Shade', // FIXME
+        'Shade Ability': 'Shade Ability', // FIXME
+        'Shingan': 'Impact oculaire',
+        'Shooting Star': 'Étoile filante',
+        'Soak Attack': 'Soak Attack', // FIXME
+        'Spear Copy': 'Spear Copy', // FIXME
+        'Spear Shade': 'Spear Shade', // FIXME
+        'Sphere Form': 'Sphere Form', // FIXME
+        'Spiritcull': 'Salve magique',
+        'Spirits of the Fallen': 'Âme héroïque',
+        'Stack': 'Stack', // FIXME
+        'Star Form': 'Star Form', // FIXME
+        'Streak Lightning': 'Éclair chargeant',
+        'Thricecull': 'Triple perforation',
+        'Thundercall': 'Drain fulminant',
+        'Turbulent Aether': 'Turbulence éthérée',
+        'Ultimate Zantetsuken': 'Goku Zantetsuken',
+        'Umbral Essence': 'Bras de Ténèbres',
+        'Whirling Zantetsuken': 'Sen Zantetsuken',
+        'Wyvern Explosion': 'Wyvern Explosion', // FIXME
+      },
+    },
+    {
+      'locale': 'ja',
+      'replaceSync': {
+        'Absolute Virtue': 'アブソリュートヴァーチュー',
+        'Arsenal Centaur': 'アーセナル・セントール',
+        'Art': 'アルト',
+        'Ball Lightning': '雷球',
+        'Cloudlarker': 'クラウドラーカー',
+        'Ivory Palm': '白き手',
+        'Orlasrach': 'オールラスラッハ',
+        'Owain': 'オーウェン',
+        'Ozma': 'オズマ',
+        'Ozmashade': 'オズマの影',
+        'Proto Ozma': 'プロトオズマ',
+        'Raiden': 'ライディーン',
+        'Relative Virtue': 'レラティブヴァーチュー',
+        'Removing combatant Proto Ozma': 'Removing combatant Proto Ozma', // FIXME
```

Comment:
I see this and other Added combatants.  I don't think you need to include the Removing combatant part, just the name (which is translated above).

Reply:
hope i catched all of them :)